bhavye/lec-5-jni/README.md

```diff
@@ -1,14 +1,15 @@
 ####################### PROCEDURE FOR MAKING A NATIVE FUNCTION ###########################
 
-################### part one (java program)##############
+################### Part one (java program)##############
 
```

Comment:
Instead of this use proper markdown for headings.

Reply:
okay.